card/lib/card/format.rb

```diff
@@ -114,15 +114,15 @@ def include_set_format_modules
       end
     end
   
-    def inclusion_defaults
+    def inclusion_defaults nested_card
       @inclusion_defaults ||= begin
-        defaults = get_inclusion_defaults.clone
+        defaults = get_inclusion_defaults(nested_card).clone
         defaults.merge! @inclusion_opts if @inclusion_opts
         defaults
       end
```

Comment:
I think I see how to fix.  in HtmlFormat:
````
def inclusion_defaults nested_card, opts={}  #send opts from nest
  defaults = super
  if !opts[:view] && !defaults[:view]
    defaults.merge :view=>(nested_card.rule(:default_html_view) || :titled )
  else
    defaults
  end
end
````
then you don't actually have to give an arg to get_inclusion_defaults